adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -186,7 +191,10 @@ - (BOOL)hasDelegateForPlacementID:(nonnull NSString *)placementID
 }
 
 - (BOOL)canRequestBannerAdForPlacementID:(nonnull NSString *)placmentID {
-  return _bannerPlacementID == nil || [_bannerPlacementID isEqualToString:placmentID];
+  if (_bannerDelegates.count > 0) {
+    return _bannerPlacementID == nil || [_bannerPlacementID isEqualToString:placmentID];
+    }
+  return YES;
```

Comment:
Can Vungle SDK now load multiple banners for different sizes? How about multiple banners for the same size? If there are no delegates, shouldn't `placementID` be `nil` and this should have passed anyways?

Reply:
@joshuajplagonera Hi joshuajplagonera, we don't support multiple banners right now. If there are no delegates, it will return YES directly, not check placementID is nil(in this case, I think the placementID should be nil). Thanks a lot.  🙂